docker/entrypoint.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/sh
 
 # Define the start command
-START_COMMAND="/usr/local/bin/sim-cli $SIMFILE_PATH"
+START_COMMAND="/usr/local/bin/sim-cli --sim-file $SIMFILE_PATH"
```

Comment:
Shouldn't this only be passed now if `$SIMFILE_PATH` is defined? If not passed, this will default to pick it from `data_dir/sim.json`

Reply:
We still need it because it looks for the config JSON in the root by default. whereas in docker, we're mounting a volume that hosts the JSON. So we need to point it to the docker volume where the sim.json is hosted and use that as a default if no SIMFILE_PATH is specified